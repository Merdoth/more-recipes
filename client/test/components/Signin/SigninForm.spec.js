import React from 'react';
import { shallow } from 'enzyme';
import expect from 'expect';
import { SigninForm } from
  '../../../components/Signin/SigninForm.jsx';

/* global jest */

const setup = () => shallow(<SigninForm />);

describe('SigninForm Component snapshot', () => {
  it('it should render the right amount of elements', () => {
    const wrapper = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
