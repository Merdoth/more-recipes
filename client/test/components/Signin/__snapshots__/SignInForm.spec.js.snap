// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SigninForm Component snapshot it should render the right amount of elements 1`] = `
<div
  className="row"
>
  <div
    className="col-md-4 offset-md-4 cover1"
  >
    <div
      className="form-deco"
    >
      <form
        className="form-signin"
      >
        <h2
          className="form-signin-heading"
        >
          Sign In
        </h2>
        <hr />
        <InputField
          label="Email Address"
          name="email"
          onChange={[Function]}
          placeholder="Enter your email address"
          required={true}
          type="email"
          value=""
        />
        <InputField
          label="Password"
          name="password"
          onChange={[Function]}
          placeholder="Enter your password"
          required={true}
          type="password"
          value=""
        />
        <Button
          className="btn btn-lg btn-primary btn-block"
          disabled={false}
          iconClass="fa-user-plus"
          name="Sign In"
          onClick={[Function]}
          type="submit"
        />
        <p
          className="new_account"
        >
          <strong>
            Create A New Account? 
          </strong>
          <Link
            replace={false}
            to="Signup"
          >
            Sign Up
          </Link>
        </p>
      </form>
    </div>
  </div>
</div>
`;
